src/components/HorizontalCard/HorizontalCard.module.scss

```diff
@@ -0,0 +1,163 @@
+@use '../../styles/utilities/_variables' as *;
+
+.horizontalCard {
+  display: flex;
+  align-items: center;
+  padding: var(--space-md);
+  cursor: pointer;
+  width: 100%;
+  overflow: hidden;
+
+  @include for-mobile-only {
+    width: 100% !important;
```

Comment:
do we need this `!important`? it will force a recalculation of the stylesheet, hurting our load perfomance

Reply:
I don't know why but the component is not taking up the full width on the small mobile, the !important here also doesn't seem to work in any way.